package.nls.json

```diff
@@ -27,8 +27,9 @@
     "configuration.java.test.config.envFile.description": "Specify the absolute path to a file containing environment variable definitions.",
     "configuration.java.test.config.sourcePaths.description": "Specify extra source paths when debugging the tests.",
     "configuration.java.test.config.preLaunchTask.description": "Specify the label of a task specified in tasks.json (in the workspace's .vscode folder). The task will be launched before the start of testing.",
+    "configuration.java.test.config.testKind.description": "Specify the targeting test framework for this test configuration. Supported values are `junit`, `testng`.",
     "configuration.java.test.config.filters.description": "Specify the test filters.",
-    "configuration.java.test.config.filters.tags.description": "Specify the tags to be included or excluded. \n\nTags having `!` as the prefix will be **excluded**. \n\nNote: It **only** takes effect on JUnit 5 tests.",
+    "configuration.java.test.config.filters.tags.description": "Specify the tags to be included or excluded. \n\nTags having `!` as the prefix will be **excluded**. \n\nNote: This setting **only** takes effect on JUnit 5 tests, you need to explicitly set the `testKind` to `junit`.",
```

Comment:
since default value is already `junit`, this statement looks a little bit strange.

Reply:
Agree, I just updated the description.